tasks/main.yml

```diff
@@ -19,6 +19,11 @@
       {{ mssql_accept_microsoft_sql_server_2019_standard_eula }}
   when: mssql_accept_microsoft_sql_server_2019_standard_eula is defined
 
+- name: Link the deprecated mssql_input_sql_file fact
+  set_fact:
+    mssql_input_sql_file: "{{ mssql_input_sql_file }}"
+  when: mssql_input_sql_file is defined
```

Comment:
?  Not sure what this does

Reply:
This is a typo, my intention was to set `mssql_post_input_sql_file: "{{ mssql_input_sql_file }}"` so that the `mssql_input_sql_file` variable that somebody might have in old playbooks work. Fixed. I've also added a debug module above to print a deprecated warning.